src/btree/bt_read.c

```diff
@@ -212,9 +213,18 @@ __page_read(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
     ref->page->block_meta = block_meta;
     ref->page->old_rec_lsn_max = block_meta.disagg_lsn;
     ref->page->rec_lsn_max = block_meta.disagg_lsn;
+
     if (instantiate_upd && !WT_IS_HS(session->dhandle))
         WT_ERR(__wti_page_inmem_updates(session, ref));
 
+    /* Reconstruct deltas*/
```

Comment:
Why move this? I'm pretty sure it's fine, I just don't understand why.

Reply:
oh, this is still wrong. It should before line 217.